### PR TITLE
Refactor some codes for consistency

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- As title, some codes need to refactored and the `\` is should be added before native PHP functions because of consistency.
- Using the `abs` function to get the correct `$absoluteValue`.
- Organize the methods/variables declaration approach.

BTW, some classes have the similar issue and it's also related PR. But this PR is too huge.

Other similar refactoring works will be in the oncoming PRs.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -95,10 +95,10 @@ public function getClientApi(): ClientApi
     public function getNode($key = null): Node
     {
         if (null === $key) {
-            $key = array_rand($this->nodes);
+            $key = \array_rand($this->nodes);
         }
 
-        if (!isset($this->nodes[$key])) {
+        if (empty($this->nodes[$key])) {
```
Comment (Techworker) [inline src/Client.php]: Why is this change necessary?

**Review by Techworker — COMMENTED** [review]
All fine, just answer the one question in the review. edit: found one more, see other review.

File: src/Client.php (hunk 1 above)
Comment (peter279k) [reply src/Client.php]: For some reasons, using the `empty` function to check whether the variable is set.
And I also think this will be readable than using the `isset` function.

You can see more details about [that](http://php.net/manual/en/function.empty.php) in official PHP doc.

Comment (Techworker) [reply src/Client.php]: I think `isset` is pretty straight forward and readable, a common contruct to check if a key in an array exists. One could use `array_key_exists` to make it more readable, but in the end the array key has to exist and the related value should be of type `Node`: both checks are there (isset + return type). So the function works as intended and using empty will not change it to the better.

Comment (peter279k) [reply src/Client.php]: Ok. It makes sense and I will revert this.

File: src/Util/TrytesUtil.php [hunk 2]
```diff
@@ -105,17 +105,15 @@ public static function asciiFromTrytes(Trytes $inputTrytes): string
         // If input length is odd, return null
         if (1 === $inputTrytes->count() % 2) {
             // TODO: we can do better than that
-            return '';
-
-            throw new Exception('not even.');
+            return null;
```
Comment (Techworker) [inline src/Util/TrytesUtil.php]: Returning null will throw a fatal error as the functions return type typehints to string

Comment (Techworker) [reply src/Util/TrytesUtil.php]: We should throw an exception, because it's clearly an error and you shouldn't proceed with anything in this case. Not sure when this will even happen but if it happens, the sent trytes are possibly not valid.

The exception message should indeed give more info than "not even", maybe sth like "Unable to derive the ascii value of the given trytes as the number of trytes is not even" or sth. like that.

File: src/Client.php (hunk 1 above)
Comment (peter279k) [reply src/Client.php]: It has been reverted.

**Review by Techworker — CHANGES_REQUESTED** [review]
See my comments here: https://github.com/iota-php/client/pull/51/files#diff-7ad51a60475cfcf8457dc2e817c2a9c2

File: src/Util/TrytesUtil.php (hunk 2 above)
Comment (peter279k) [reply src/Util/TrytesUtil.php]: @Techworker, thank you for your reply. I will remove the `return null` and add the `exception` message.

**Comment by peter279k** [comment]
@Techworker, thank you for your request changes, I've enhanced that and I appreciate your review work.

I also fix the conflicts from the master branch.

**Review by Techworker — CHANGES_REQUESTED** [review]
Sorry, this is not valid, it's missing the `new` keyword. Please test your code before submitting, thx!

Also, I think we should throw an `\InvalidArgumentException`

A simple local test would have shown you the problem, not even a PHPUnit necessary:

```
use IOTA\Type\Trytes;
use IOTA\Util\TrytesUtil;

require_once 'wherever/autoload.php';
TrytesUtil::asciiFromTrytes(new Trytes('A'));
```

**Comment by peter279k** [comment]
@Techworker, thank you for your reply. I almost forgot the `new` keyword :smile:.

**Review by Techworker — APPROVED** [review]
I think we can give this a go now, just fix the conflicts.

**Comment by peter279k** [comment]
@Techworker, thank you for your reply. I've merged the master branch into this branch :+1:.

**Review by Techworker — APPROVED** [review]

**Comment by peter279k** [comment]
The conflicts still happen after merging the PR #57.

I will fix that and don't be worried about that :smile: .
